templates/include/glean/histogram.1.parquetmr.txt

```diff
@@ -0,0 +1,14 @@
+optional group range (TUPLE) {
+  required double lo;
+  required double hi;
+}
+optional inte64 bucket_count;
+optional binary histogram_type (UTF8);
+optional group values (MAP) {
+  repeated group key_value {
+    required binary key (UTF8);
+    optional int64 value;
```

Comment:
What is the size here on the client? We do see lots of very large histogram values on desktop.

Reply:
We haven't implemented histograms yet, but I think it's reasonable to assume 64 bit like this.